build.py

```diff
@@ -401,44 +409,92 @@ def run_ci_historic_benchmark():
     step_end()
 
 if build_wasm:
-    step_start("Building the wasm crate")
-    # wasm-pack can't build for web and node in the same build, so need to run twice.
-    # Hopefully not needed if https://github.com/rustwasm/wasm-pack/issues/313 lands.
-    build_flag = "--release" if build_type == "release" else "--dev"
-
-    wasm_pack_args = ["wasm-pack", "build", build_flag]
-    web_build_args = ["--target", "web", "--out-dir", os.path.join(wasm_bld, "web")]
-    node_build_args = [
+    env = get_wasm_env()
```

Comment:
This is to put the 'default' directories where 'prereqs.py --install' downloads the utilities to on the end of the PATH. (It will still pick up any versions you have earlier on your path if manually installled).